lua/telescope/_extensions/packer.lua

```diff
@@ -30,6 +30,29 @@ local setup = function(opts)
   end
 end
 
+local function opener()
+  local sysname = vim.loop.os_uname().sysname
+  if sysname == "Darwin" then
+    return 'open'
+  elseif sysname == "Windows" then
+    return "explorer"
+  elseif vim.fn.executable("wslview") then
+    return 'wslview'
+  elseif vim.fn.executable("xdg-open") then
+    return 'xdg-open'
+  else
+    return nil
+  end
+end
+
+local function get_url(path)
+  local url = vim.fn.trim(vim.fn.system(string.format("git -C %s ls-remote --get-url", path)))
+  if string.sub(url, 1, 6) == 'ssh://' then
+    url, _ = string.gsub(url, [[^ssh://[^/]+/]], 'https://github.com/')
```

Comment:
not every `ssh://` points to github.com, we can actually get the real url from that

Reply:
e.g. if we have

```
local url = "ssh://git@gitlab.com:gitlab-org/gitlab.git"
url = string.gsub(url, [[^ssh://[^/]+/]], 'https://github.com/')
print(url)
```

i would expect `https://gitlab.com/gitlab-org/gitlab` but i currently get `https://github.com/gitlab.git` which is completely wrong

Reply:
I see.
I'll fix it

Reply:
done.